src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -89,7 +91,13 @@ public Future<JsonObject> post(String endpoint, JsonObject recordData) {
   }
 
   public Future<Void> delete(String endpointById) {
-    logger.debug(CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpointById);
+    if (logger.isDebugEnabled()) {
+//      CALLING_ENDPOINT_MSG = "Sending {} {}";
+//      logger.debug(CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpointById);
+      logger.debug("Tying to delete with using endpoint: {}", endpointById);
+      logger.debug("Sending 'DELETE {}'", endpointById);
```

Comment:
logger.debug("Tying to delete with using endpoint: {}", endpointById); - this is preferable